crates/bevy_core_pipeline/src/prepass/node.rs

```diff
@@ -50,6 +50,8 @@ impl ViewNode for PrepassNode {
     ) -> Result<(), NodeRunError> {
         let view_entity = graph.view_entity();
 
+        let _ = view_depth_texture.is_first_write();
```

Comment:
I suppose the prepass node should decide whether to load or clear depth using this result, technically.

Reply:
i think i disagree. if a node is added before the prepass (and calls is_first_write) then the prepass should not disregard that node's output.

Reply:
Aren't we agreeing? Currently the prepass unconditionally clears, but it probably shouldn't.

Reply:
er. yes i agree with your comment, i disagree with the code :)